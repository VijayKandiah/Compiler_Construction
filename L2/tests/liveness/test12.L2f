(:myF
  0 0

  %myVar <- :myF2

  call %myVar 0

  return
)
