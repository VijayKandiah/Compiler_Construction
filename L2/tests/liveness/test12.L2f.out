(
(in
(r12 r13 r14 r15 rbp rbx)
(myVar r12 r13 r14 r15 rbp rbx)
(r12 r13 r14 r15 rax rbp rbx)
)

(out
(myVar r12 r13 r14 r15 rbp rbx)
(r12 r13 r14 r15 rax rbp rbx)
()
)

)
